vcd/resource_vcd_vapp_vm.go

```diff
@@ -1777,6 +1641,24 @@ func networksToConfig(networks []interface{}, vdc *govcd.Vdc, vapp govcd.VApp, v
 	return networkConnectionSection, nil
 }
 
+// isItVappOrgNetwork checks if it is an vApp Org network (not vApp Network)
+func isItVappOrgNetwork(vAppNetworkName string, vapp govcd.VApp) (bool, error) {
+	vAppNetworkConfig, err := vapp.GetNetworkConfig()
+	if err != nil {
+		return false, fmt.Errorf("error getting vApp networks: %s", err)
+	}
+
+	for _, networkConfig := range vAppNetworkConfig.NetworkConfig {
+		if networkConfig.NetworkName == vAppNetworkName &&
+			!govcd.IsVappNetwork(networkConfig.Configuration) {
+			log.Printf("[TRACE] vApp Org network found: %s", vAppNetworkName)
+			return true, nil
+		}
+	}
+
+	return false, fmt.Errorf("configured vApp Org network isn't found: %s", vAppNetworkName)
+}
+
```

Comment:
What will happen with #541 given the above block is now here?

Reply:
closed

Reply:
Strange, it shows open for me and pending for my review :)

Reply:
will be closed, then this PR will have success merge